Ladybird/Qt/Tab.cpp

```diff
@@ -893,15 +894,28 @@ void Tab::resizeEvent(QResizeEvent* event)
 
 void Tab::update_hover_label()
 {
+    m_hover_label->setText(QFontMetrics(m_hover_label->font()).elidedText(m_hover_label->text(), Qt::ElideRight, width() / 2));
     m_hover_label->resize(QFontMetrics(m_hover_label->font()).boundingRect(m_hover_label->text()).adjusted(-4, -2, 4, 2).size());
-    auto hover_label_height = height() - m_hover_label->height() - 8;
+
+    auto hover_label_height = height() - m_hover_label->height();
     if (m_find_in_page->isVisible())
-        hover_label_height -= m_find_in_page->height() - 4;
+        hover_label_height -= m_find_in_page->height() - 8;
+
+    QPoint mousePosition = mapFromGlobal(QCursor::pos());
+    if (mousePosition.y() >= hover_label_height && mousePosition.x() <= width() / 2)
```

Comment:
This doesn't seem to update the label position when mousing over an already-visible label:

https://github.com/user-attachments/assets/ab37c804-b309-48c8-8182-30562eaa827b

Would `if (m_hover_label->underMouse() && m_hover_label->x() == 0)` work better here?

Reply:
Thank you for the suggestion, works great